submissions/TatyanaMolchanova/js-dom/script.js

```diff
@@ -0,0 +1,103 @@
+const content = {
```

Comment:
Also, don't forget about using not so generic names of your variables. `animalsData` would be better and simpler to understand

Reply:
Ok, I changed content to animalsData.